src/obb_demo/boardground.cljs

```diff
@@ -47,13 +47,22 @@
   (let [dir (element/element-direction element)]
     (first (name dir))))
 
+(defn- process-unit-name
+  "If the unit name needs transformations, does it to match the resources
+  on the main site."
+  [unit-name]
+  (if (= unit-name "star")
+    "flag"
+    unit-name))
```

Comment:
Great idea!
[Updated the commit with the issue](https://github.com/orionsbelt-battlegrounds/obb-rules/pull/35/commits/6eafa97690034b6e4be902a2993e57826cde5f71).